src/test/java/seedu/address/logic/parser/EditCommandParserTest.java

```diff
@@ -235,15 +237,15 @@ public void parseTutorials_validInput_success() throws Exception {
         List<String> tutorialsStrings = argMultimap.getAllValues(PREFIX_TUTORIAL);
 
         if (!tutorialsStrings.isEmpty()) {
-            List<Tutorial> tutorialList = ParserUtil.parseTutorials(tutorialsStrings);
-            descriptorBuilder.withTutorials(tutorialList);
+            Set<Tutorial> tutorialSet = ParserUtil.parseTutorials(tutorialsStrings);
+            descriptorBuilder.withTutorials(tutorialSet);
         }
 
-        // Replace the following with actual expected values if needed
         EditPersonDescriptor expectedDescriptor = new EditPersonDescriptorBuilder()
-            .withTutorials(List.of(new Tutorial("01"), new Tutorial("02"), new Tutorial("03")))
+            .withTutorials(new HashSet<>(List.of(new Tutorial("01"), new Tutorial("02"), new Tutorial("03"))))
             .build();
 
         assertEquals(expectedDescriptor, descriptorBuilder.build());
     }
+
```

Comment:
LGTM! but i think can add invalid tutorial value test cases to increase the code coverage.